configs/components/fesom/fesom-2.1.yaml

```
@@ -63,7 +63,9 @@ pool_dir: "${computer.pool_directories.pool}"
 
 choose_computer.name:
     ollie:
-        pool_dir: "${computer.pool_directories.pool}/../clidyn/"
+        pool_dir: "${computer.pool_directories.pool}/FESOM/"
+        climate_data_dir: "${pool_dir}/hydrography/"
+        forcing_data_dir: "${computer.pool_directories.projects}/clidyn/forcing/"
```

```suggestion
        pool_dir: "${computer.pool_directories.pool}/../clidyn/"
```
I suggest to bring back the old configuration as it is the default provided by FESOM developets and then tested with the new `fesom-2.1` runscript I added. This mesh is unrotated in contrast with the one you added on your first commit.

---

Additionally, the new runscript will also work in mistral (and in the future in Levante) because fesom-2.1 already handles correctly the pools there too.